src/commands/appSettings/connectionSettings/getLocalConnectionSetting.ts

```diff
@@ -21,11 +21,17 @@ export interface IResourceResult {
     connectionString: string;
 }
 
-export async function getStorageConnectionString(context: IStorageAccountWizardContext): Promise<IResourceResult> {
+export async function getStorageConnectionString(context: IStorageAccountWizardContext & { useStorageEmulator?: boolean }): Promise<IResourceResult> {
+    if (context.useStorageEmulator) {
+        return {
+            name: 'AZURITE',
```

Comment:
I feel like "AZURITE" as a name isn't ideal because the same name is used when you deploy to Azure and you obviously can't use Azurite in Azure. I'd pick a more generic name that makes sense for both local and remote scenarios

Some random ideas:

- connection1_STORAGE
- {randomHex}_STORAGE (for example a8b382_STORAGE)
- {functionName}Connection (for example blobTrigger1Connection)
- storageConnection1

Reply:
To be clear: my suggestions were for the end result of the app setting name, not necessarily this particular "name" variable as I'm not exactly sure how it's used.

Reply:
Going with randomHex because it's easiest and don't need to keep track of duplicates or anything like that,